static/src/stylesheets/module/facia-garnett/_item.scss

```diff
@@ -142,6 +142,13 @@ $fc-item-gutter: $gs-gutter / 4;
 // Wraps all media elements
 .fc-item__media-wrapper {
     box-sizing: border-box;
+
+    figure {
```

Comment:
As part of the Captions change I changed all image elements to be wrapped in a <figure> regardless of whether they are in a slideshow or not.

this was done for 2 reasons:
 1. Keeps the CSS simple, the idea was that we wouldn't need to have seperate image styles for Slideshows
 2. Allows potentially adding captions to any image on the page even if they are not a slideshow. The pressed model already kind of supports this.